client/src/locales/strings.json

```
@@ -6,6 +6,9 @@
     },
     "submitButton": {
       "text": "Enviar"
+    },
+    "SearchInput": {
+      "placeholder": "Búsqueda..."
```

Nit:
```suggestion
      "placeholder": "Buscar"
```

Una boludez pero lo mando porque hice una quick-search por varias webs que tengo abiertas y todas usan `Buscar` jaja pero me da lo mismo la verdad.

---

Si! me parece bien la verdad, yo nomas guiándome por el figma, pero queda mejor el Buscar

---

@Santisantt01 he cambiado el placeholder, si te animas a cambiarlo luego en el figma golazo!